.travis.yml

```diff
@@ -29,4 +29,4 @@ branches:
 before_install:
   - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install openssl; ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/; ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/; fi
 script:
-  - ./build.sh
+  - ./build.sh |tee /dev/null
```

Comment:
Yeah Travis has limits on how much log output it'll take... and MVC's build produces waaaaaay too much output 😄